consensus/istanbul/core/handler.go

```diff
@@ -121,11 +121,17 @@ func (c *core) handleEvents() {
 					//c.backend.Gossip(c.valSet, p)
 				}
 			}
-		case _, ok := <-c.timeoutSub.Chan():
-			if !ok {
+		case ev, ok := <-c.timeoutSub.Chan():
+			if !ok || ev.Data == nil {
+				logger.Error("Drop an empty message from timeout channel")
+				return
+			}
+			data, ok := ev.Data.(timeoutEvent)
+			if !ok || data.nextView == nil {
+				logger.Error("Invalid message from timeout channel", "msg", ev.Data)
```

Comment:
Does printing `ev.Data` works well? It prints bytes?
```suggestion
				logger.Error("Invalid message from timeout channel", "msg", ev.Data)
```

Reply:
@KimKyungup If `ev.Data` is string, data is printed as a string. If `ev.Data` is bytes, data is printed as bytes.
Actually, this code should not be reached by any context since `c.timeoutSub.Chan` only subscribes `timeoutEvent` and `ev.Data` would be `nil` if `data.nextView` is nil.